cyclonedx_py/client.py

```diff
@@ -235,6 +235,10 @@ def get_arg_parser(*, prog: Optional[str] = None) -> argparse.ArgumentParser:
             '-pb', '--purl-bom-ref', action='store_true', dest='use_purl_bom_ref',
             help="Use a component's PURL for the bom-ref value, instead of a random UUID"
         )
+        arg_parser.add_argument(
+            "-omit", "--omit", dest="omit", action="append",
+            help="Omit specified items when using Poetry or PipEnv (currently supported is dev)",
```

Comment:
:exclamation: help text does not declare allowed values properly. 
Let's mark this as an open topic and concentrate on the implementations first. 

FYI: CLI and argparse might change to another implementation soon.
don't waste time now, we can discuss this at the feature finalization phase.

Reply:
My assumption was that *currently supported is dev* is enough to get started.

Reply:
i guess @madpah will review the existing CLI and rework it anyway.

Reply:
lets leave this open.
if we go with Click8 then the help text generator will take over here.

Reply:
we will fix this during the CLI/click8/subcomamnd rework